lib/biokbase/catalog/db.py

```diff
@@ -1337,49 +1459,52 @@ def update_db_1_to_2(self):
 
             # make sure everything has the dynamic service flag
             if not 'dynamic_service' in m['info']:
-                self.modules.update_one(
+                modules_collection.update_one(
                     {'_id': m['_id']},
                     {'$set': {'info.dynamic_service': 0}})
 
             if m['current_versions']['release']:
                 if not 'dynamic_service' in m['current_versions']['release']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.release.dynamic_service': 0}})
 
             if m['current_versions']['beta']:
                 if not 'dynamic_service' in m['current_versions']['beta']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.beta.dynamic_service': 0}})
 
             if m['current_versions']['dev']:
                 if not 'dynamic_service' in m['current_versions']['dev']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.dev.dynamic_service': 0}})
 
         # also ensure the execution stats fields have correct names
-        self.exec_stats_apps.update_many({'avg_queue_time': {'$exists': True}},
+        exec_stats_apps_colleciton = db[MongoCatalogDBI._EXEC_STATS_APPS]
+        exec_stats_apps_colleciton.update_many({'avg_queue_time': {'$exists': True}},
                                          {'$rename': {'avg_queue_time': 'total_queue_time',
                                                       'avg_exec_time': 'total_exec_time'}})
-        self.exec_stats_users.update_many({'avg_queue_time': {'$exists': True}},
+        exec_stats_apps_colleciton.update_many({'avg_queue_time': {'$exists': True}},
```

Comment:
This looks like a C&P error

Reply:
Can you run through the changes here one more time and ensure there aren't any other errors like this, where the collection is switched (assuming I'm understanding this correctly)

Reply:
👍

Reply:
Does that mean you double checked all the collection changes where `self` was removed?

Reply:
yes, I did

Reply:
:+1: